tests/credential_plugins_test.py

```diff
@@ -137,13 +137,18 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
+    def test_dsv_import(self) -> None:
         from awx_plugins.credentials.dsv import SecretsVault  # noqa
         # assert this module as opposed to older thycotic.secrets.vault
         assert SecretsVault.__module__ == 'delinea.secrets.vault'
 
-    def test_tss_import(self):
-        from awx_plugins.credentials.tss import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret  # noqa
+    def test_tss_import(self) -> None:
+        from awx_plugins.credentials.tss import (  # noqa
+            DomainPasswordGrantAuthorizer,
+            PasswordGrantAuthorizer,
+            SecretServer,
+            ServerSecret,
+        )
```

Comment:
Plz keep the original formatting. Formatting changes must never be mixed with the functional ones, due to the confusing perception this causes by making it look like these lines have *something* related to the goal advertised in the PR title, when there's none.

Reply:
Ah, okay! Thank you!